moz_telemetry/io_modules/decoders/moz_telemetry/ping.lua

```diff
@@ -323,26 +337,35 @@ local function remove_objects(msg, doc, section, objects)
 end
 
 
+local function validate_schema(hsr, msg, doc, version)
```

Comment:
re: vacuous schema - the behaviour is fine as-is, I'd just like to see a descriptive comment for future readers about how it is used.

Sounds good re: schema reloading.